src/components/__tests__/HierarchicalMenu.js

```diff
@@ -478,11 +482,10 @@ it('calls the Panel mixin with `items.length`', async () => {
 
   expect(mapStateToCanRefine()).toBe(true);
 
-  await wrapper.setData({
-    state: {
-      items: [],
-    },
-  });
+  // should've used wrapper.setData({ state: { items: [] }})
+  // but https://github.com/vuejs/vue-test-utils-next/issues/766
+  wrapper.vm.state.items = [];
```

Comment:
note to also revert this once https://github.com/vuejs/vue-test-utils-next/issues/766 is released